README.md

```diff
@@ -38,6 +38,32 @@ OpenColorIO is free and open source software ([LICENSE](LICENSE)), and
 one of several projects actvively sponsored by the ASWF
 ([Academy Software Foundation](https://www.aswf.io/)).
 
+OpenColorIO Project Mission
+---------------------------
+
+The OpenColorIO project is committed to providing an industry standard solution 
+for highly precise, performant, and consistent color mangement across digital 
+content creation applications and pipelines.
```

Comment:
I feel like this should be following by a clarifying statement on what the "industry" is, and perhaps clarifying the critical role OCIO currently plays in the industry (and thus why these goals are important), but I was struggling with the right wording...

Reply:
maybe I should just replace "industry standard" with "best in class"

Reply:
Keeping as-is for now.